### ISSUE TITLE
Can particle be in a discourse labeled dependant of a conjunction?

### ISSUE THREAD

**Issue by lauma** [UniversalDependencies/docs#595]
Let's consider following sentence in Latvian:
_Viņš nesteidzās, kaut arī bija vēls._
He didn't hurry even though [it] was late.

I'm struggling to determine correct UD analysis for _kaut arī_. Older grammar books for Latvian consider it together with several others like _lai arī_, _lai gan_, _kaut gan_ to be basically a multiword coordination conjunction. However, for all kinds of computational purposes to consider this as one token is very cumbersome*, thus, Latvian Treebank and all tools related considers _kaut_ to be a particle, and _arī_ to be a conjunction. Also, this fits well with analysis when _kaut_ or _arī_ is used alone. To carry on the traditional notion that these two tokens are something of a unit together, Latvian Treebank makes _kaut_ to be dependant of _arī_.

However, the new UD validator currently forbids conjunctions to have dependants. Would making an exception for dependants labeled `discourse` make sense? Would analizing this as conjunction with `fixed` dependant be more in lines with UD intentions? Is there any other options I didn't think of?

____________________________
*) not only _kaut_ can be used as an standalone particle, it also happens to be homonymous with a verb _kaut_ (slay, slaugther) infinitive form, thus a sentence _viņi nolēma kaut arī govi_ (they decided to slaughter a cow, too) is a completely valid sentence which most certanly must be tokenized in 4 tokens, not 3.

**Comment by dan-zeman** [comment]
I would definitely keep _kaut_ as a separate token; words with spaces should be used very sparingly.

Using `discourse` does not seem appropriate to me; [according to the guidelines](http://universaldependencies.org/u/dep/discourse.html), `discourse` would have to be attached to the head of the clause (and it would have to be something which is not clearly linked to its structure, which is not the case here).

I think the right relation here is `fixed`. The validator should accept it (but you will be forced to make _kaut_ the pseudohead of the fixed expression, not _arī_).

**Comment by sylvainkahane** [comment]
Is _kaut_ a CCONJ or an SCONJ? Can you explain how you decide?

**Comment by lauma** [comment]
Oh, I got corrected, in the abovementioned case _kaut_ is the subordination conjunction and _arī_ is the particle, not the opposite way.

Anyway, I like the solution with `fixed`, I'll go with it, unless someone objects.

@sylvainkahane _kaut_ is traditionally considered to be subordination conjunction, thus, `SCONJ` in UD. The newest comprehensive grammar book (Latviešu valodas gramatika, 2013) mentions that the distinction for this particular word is somewhat complicated, but as we currently don't have resources to make indepth research about this particular conjunction, we just go with the tradition :)
